types/paseto-tests.ts

```diff
@@ -132,5 +132,7 @@ import * as paseto from './index.d'
       now: new Date(),
       subject: 'string',
     })
+
+    b['subject']
```

Comment:
Can we add the unknown assert here and use a claim that isn't a known one? I knew subject isn't but it could be mistaken for sub (subject). Let's just use "arbitrary claim" here.

Reply:
Would you like that in addition to the known `'subject'` claim or in place of?

Reply:
In place of.